lib/rmg/Zone.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Zone.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#include "../GameConstants.h"
+#include "float3.h"
+#include "../int3.h"
+#include "CRmgTemplate.h"
+#include "RmgArea.h"
+#include "RmgPath.h"
+#include "RmgObject.h"
+
+#define RMG_DUMP
```

Comment:
Do we need it now? I mean dumping enabled after merge? I usually disable such things before merge to avoid performance impact.